### PR TITLE
fix: image sizing and boosted tag

### PR DESCRIPTION
**Pull request by ayushtom**

### CODE REVIEW

File: app/quest/[questPage]/page.tsx [hunk 1]
```diff
@@ -17,8 +17,27 @@ export async function generateMetadata(
 
   try {
     const data = await fetchQuestData(questId);
+    console.log({ data });
 
     if (data?.name) {
+      console.log({
```
Comment (fricoben) [inline app/quest/[questPage]/page.tsx]: to delete

File: components/quests/quest.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import styles from "@styles/quests.module.css";
 import { CDNImg } from "@components/cdn/image";
 import QuestCard from "./questCard";
 
-const BOOSTED_QUESTS = [23];
+const BOOSTED_QUESTS = [23, 104];
```
Comment (fricoben) [inline components/quests/quest.tsx]: We need to fin something else for that

**Review by fricoben — CHANGES_REQUESTED** [review]
console log

File: app/quest/[questPage]/page.tsx (hunk 1 above)
Comment (ayushtom) [reply app/quest/[questPage]/page.tsx]: done

File: components/quests/quest.tsx (hunk 2 above)
Comment (ayushtom) [reply components/quests/quest.tsx]: done

Comment (ayushtom) [reply components/quests/quest.tsx]: added new api endpoint here -
which will return array like this 
`
[
    13,
    2,
    6,
    23,
    1,
    13
]
`

File: services/apiService.ts [hunk 3]
```diff
@@ -112,3 +112,12 @@ export const getPendingBoostClaims = async (addr: string) => {
     console.log(err);
   }
 };
+
+export const getBoostedQuests = async () => {
+  try {
+    const response = await fetch(`${baseurl}/get_boosted_quests`);
+    return await response.json();
+  } catch (err) {
+    console.log(err);
```
Comment (fricoben) [inline services/apiService.ts]: Same here. 

IF possible change it everywhere

File: services/apiService.ts [hunk 4]
```diff
@@ -112,3 +112,12 @@ export const getPendingBoostClaims = async (addr: string) => {
     console.log(err);
```
Comment (fricoben) [inline services/apiService.ts]: Same comment that on the other PR.

console.log("Error while ...", error);

**Review by fricoben — CHANGES_REQUESTED** [review]
little change and we're good

File: services/apiService.ts (hunk 4 above)
Comment (ayushtom) [reply services/apiService.ts]: done for others in boost v2 PR

File: services/apiService.ts (hunk 3 above)
Comment (ayushtom) [reply services/apiService.ts]: done

Comment (fricoben) [reply services/apiService.ts]: Then can you delete the `console.log(err)` here and everywhere you find it? 

Cause you print two times now lol.

**Review by fricoben — CHANGES_REQUESTED** [review]
Inattention errors ;p

File: services/apiService.ts (hunk 3 above)
Comment (ayushtom) [reply services/apiService.ts]: fixed

**Review by fricoben — APPROVED** [review]
lgtm

**Pull request merged**